### PR TITLE
update usafacts to use new lag style

### PR DESCRIPTION
**Pull request by sgsmob**
### Description
Update `usafacts` validation to use the conventions introduced in #915.

### Fixes 
- Partial step for #725

### CODE REVIEW

**Review by jingjtang — APPROVED** [review]
lgtm

File: ansible/templates/usafacts-params-prod.json.j2 [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "common": {
       "data_source": "usa-facts",
       "span_length": 14,
-      "end_date": "latest",
+      "end_date": "today",
```
Comment (krivard) [inline ansible/templates/usafacts-params-prod.json.j2:22]:
```suggestion
      "end_date": "today-2",
```

File: usafacts/params.json.template [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "common": {
       "data_source": "usa-facts",
       "span_length": 14,
-      "end_date": "latest",
+      "end_date": "today",
```
Comment (krivard) [inline usafacts/params.json.template:22]:
```suggestion
      "end_date": "today-2",
```

**Review by krivard — COMMENTED** [review]
Update reflecting min_lag pulled from metadata (which has some signals at min_lag 1, some signals at min_lag 2, guessing we'd rather use the worst case there?)

**Comment by sgsmob** [comment]
> Update reflecting min_lag pulled from metadata (which has some signals at min_lag 1, some signals at min_lag 2, guessing we'd rather use the worst case there?)

Thank you for making these suggestions on this PR and the others.  Done

**Pull request merged**